### PR TITLE
BaSP-MR-169: Login layout

### PR DESCRIPTION
**Pull request by ghost**
Make the login screen based on the figma model. Using React-hook-form and JOI for validations.

### CODE REVIEW

File: src/Validations/login.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Joi from 'joi';
+
+const loginValidation = Joi.object({
+  email: Joi.string().regex(/.+/).label('Email').messages({
```
Comment (fernandofernandezuy) [inline src/Validations/login.jsx]: Usea a valid regex email please

**Review by Federico-Peralta — COMMENTED** [review]
Try to change the background color of the form

File: src/Components/Login/login.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  width: 55%;
+  height: 5rem;
+}
+
+.eyeButton {
+  display: flex;
+  align-self: center;
+  border: none;
+  background-color: transparent;
+  scale: 1.7;
```
Comment (lorenzomauro59) [inline src/Components/Login/login.module.css]: I would leave the eye with its original size to mark consistency with the others used in admin and member

**Review by lorenzomauro59 — APPROVED** [review]
Gooood job Dani! I only leave you that aesthetic suggestion

**Review by Federico-Peralta — APPROVED** [review]

**Comment by araefedg** [comment]
💯👌😊

**Pull request merged**